Libraries/LibHTTP/Job.cpp

```diff
@@ -268,7 +268,7 @@ void Job::on_socket_connected()
                     content_length = length;
             }
 
-            did_progress(content_length, m_received_size);
+            deferred_invoke([this, content_length](auto&) { did_progress(content_length, m_received_size); });
```

Comment:
Have you tested https with this change?
I vaguely remember having problems with it

Reply:
Only briefly - loading pages seems to work okay in the browser and some simple `pro` tests went through okay. Is there any specific circumstance I should be looking at?